### ISSUE TITLE
init.sh exits erroneously

### ISSUE THREAD

**Issue by vsagarv** [ajithnn/rupaka#1]
**OS**: MacOS BigSur 11.6.7

**Issue**: The example provided in the README, generates ouput-d.json correctly based on example-1.cfg/vld but init.sh exits erroneously as shown here:

```
rupaka % docker run --rm -i --mount type=bind,src="$(pwd)"/examples,dst=/app/rupaka/examples -t slashring/rupaka:v0.0.5 -o /app/rupaka/examples/ouput-d.json -c /app/rupaka/examples/example-1.cfg -v /app/rupaka/examples/example-1.vld

/app/rupaka/init.sh: line 3: "Generated: command not found
```
That "Generated" word seems to be coming from this line in app/Main.hs
```
print "Generated Json Configurations"
```
In turn, init.sh, due to back-quote command substitution, tries to execute 'Generated' as a command.

**Question**: It doesn't appear that rupaka/rupaka would generate a command string. Shouldn't that command substitution be removed and just run as follows?
```
#! /bin/bash

/app/rupaka/rupaka "$@"
```

**Typo**: ouput-d.json in the example should be output-d.json. Took a few seconds to figure why the file wasn't showing up when  I was looking for output-d.json.

**Comment by ajithnn** [comment]
I have seen this , should go away with removal of command substitution , in an older version there was a command that was replaced by this and I have missed the command substitution. 

Typo:  Will fix the Readme. Thanks

**Comment by ajithnn** [comment]
fixed in b84670db71cbed89d2bb145bede52746ee31be7f
